docs/pages/access-controls/access-request-plugins/ssh-approval-opsgenie.mdx

```diff
@@ -0,0 +1,114 @@
+---
+title: Access Requests with Opsgenie
+description: How to set up Teleport's Opsgenie plugin for privilege elevation approvals.
+---
+
+With Teleport's Opsgenie integration, engineers can access the infrastructure
+they need to resolve alerts quickly—without longstanding admin permissions
+that can become a vector for attacks.
+
+Teleport's Opsgenie integration allows you to treat Teleport Role Access
+Requests as Opsgenie alerts, notify the appropriate on-call team, and
+approve or deny the requests via Teleport. You can also configure the plugin to
+approve Role Access Requests automatically if the user making the request is on
+the on-call team for a service affected by an alert.
+
+This guide will explain how to set up Teleport's Access Request plugin for
+Opsgenie.
+
+(!docs/pages/includes/plugins/diagram.mdx api="Opsgenie" action="Listen for Access Requests\nModify Access Requests" messages="Send notifications\nGet alert status" !)
+
+## Prerequisites
+
+(!docs/pages/includes/commercial-prereqs-tabs.mdx!)
+
+- An Opsgenie account with the ability to create API keys with the 'read' and,
+  'create and update' access rights.
+
+  You can create API keys by going to the Opsgenie settings page. Navigating to
+  the 'APP settings' section and selecting API key management.
+
+- This feature is currently only available for Cloud isntances.
+
+- (!docs/pages/includes/tctl.mdx!)
+
+## Step 1/5. Create services
+
+To demonstrate the Opsgenie plugin, create an Opsgenie team named
+`teleport-access-request-notifications`.
+
+We will configure the Opsgenie plugin to create an alert for the
+`teleport-access-request-notifications` team when certain users
+create an Access Request.
+
+## Step 2/5. Define RBAC resources
```

Comment:
Like I mentioned below, in this section we should also explain how to setup a role with annotations that control which schedules we will send the request to, and give an example role.